ci/daily-snapshot.yml

```diff
@@ -32,13 +32,20 @@ jobs:
         prefix=$(head -1 LATEST | awk '{print $2}' | sed -e 's/\([^-]*\).*/\1/')
         release=$(./release.sh snapshot HEAD $prefix | awk '{print $2}')
 
-        setvar is_release true
-        setvar trigger_sha "$(branch_sha)"
-        setvar release_sha "$(branch_sha)"
-        setvar release_tag "$release"
-        setvar split_release_process true
+        if ! curl -u $AUTH https://digitalasset.jfrog.io/artifactory/api/storage/assembly/daml/$release --fail; then
```

Comment:
Perhaps this is too corner-casey, but could we check for a 404 error specifically, so that other unrelated artifactory issues won't cause false negatives?